kernel/shop/customerorderview.php

```diff
@@ -16,6 +16,13 @@
 $tpl = eZTemplate::factory();
 
 $Email = urldecode( $Email );
+
+// workaround because it seems not possible to get an urlencoded "+" character accross $Params
+if ( $http->hasGetVariable( "email" ) )
+{
+    $Email = $http->getVariable( "email" );
+}
+
```

Comment:
@pbek. no, the patch is good. forget my comment. just though that now [this line](https://github.com/pbek/ezpublish-legacy/blob/master/kernel/shop/customerorderview.php#L10)
could throw a php notice or something now $Params['Email'] won't be set in this new urls.